File: app/src/main/java/ru/prsolution/winstrike/mvp/apimodels/Room.java

package ru.prsolution.winstrike.mvp.apimodels;

import java.io.Serializable;
import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Room implements Serializable {

  @SerializedName("name")
  @Expose
  private String name;
  @SerializedName("public_id")
  @Expose
  private String publicId;
  @SerializedName("room_layout_pid")
  @Expose
  private String roomLayoutPid;

  @SerializedName("description")
  @Expose
  private String description;

  @SerializedName("image_url")
  @Expose
  private String imageUrl;

  @SerializedName("locale")
  @Expose
  private String locale;

  @SerializedName("usual_description")
  @Expose
  private String usualDescription;

  @SerializedName("usual_image_url")
  @Expose
  private String usualImageUrl;

  @SerializedName("vip_description")
  @Expose
  private String vipDescription;

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getPublicId() {
    return publicId;
  }

  public void setPublicId(String publicId) {
    this.publicId = publicId;
  }

  public String getRoomLayoutPid() {
    return roomLayoutPid;
  }

  public void setRoomLayoutPid(String roomLayoutPid) {
    this.roomLayoutPid = roomLayoutPid;
  }

  public String getDescription() {
    return description;
  }

  public void setDescription(String description) {
    this.description = description;
  }

  public String getImageUrl() {
    return imageUrl;
  }

  public void setImageUrl(String imageUrl) {
    this.imageUrl = imageUrl;
  }

  public String getLocale() {
    return locale;
  }

  public void setLocale(String locale) {
    this.locale = locale;
  }

  public String getUsualDescription() {
    return usualDescription;
  }

  public void setUsualDescription(String usualDescription) {
    this.usualDescription = usualDescription;
  }

  public String getUsualImageUrl() {
    return usualImageUrl;
  }

  public void setUsualImageUrl(String usualImageUrl) {
    this.usualImageUrl = usualImageUrl;
  }

  public String getVipDescription() {
    return vipDescription;
  }

  public void setVipDescription(String vipDescription) {
    this.vipDescription = vipDescription;
  }
}

File: app/src/main/java/ru/prsolution/winstrike/mvp/apimodels/Room.kt
package ru.prsolution.winstrike.mvp.apimodels

import com.google.gson.annotations.SerializedName
import ru.prsolution.winstrike.R
import java.io.Serializable

class Room : Serializable {

	@SerializedName("name")
	var name: String? = null

	@SerializedName("public_id")
	var publicId: String? = null

	@SerializedName("metro")
	var metro: String? = null

	@SerializedName("room_layout_pid")
	var roomLayoutPid: String? = null

	@SerializedName("description")
	var description: String? = null

	@SerializedName("image_url")
	var imageUrl: String? = null

	var imageDefault: Int = R.drawable.vip

	@SerializedName("locale")
	var locale: String? = null

	@SerializedName("usual_description")
	var usualDescription: String? = null

	@SerializedName("usual_image_url")
	var usualImageUrl: String? = null

	@SerializedName("vip_image_url")
	var vipImageUrl: String? = null

	@SerializedName("vip_description")
	var vipDescription: String? = null
}

